src/server/raftkv.rs

```diff
@@ -367,6 +385,12 @@ impl<S: RaftStoreRouter<RocksEngine>> Engine for RaftKv<S> {
                     ASYNC_REQUESTS_DURATIONS_VEC
                         .write
                         .observe(begin_instant.elapsed_secs());
+                    if let Some(tag) = tag {
+                        ASYNC_WRITE_DURATIONS_VEC
+                            .get(tag)
+                            .observe(begin_instant.elapsed_secs());
```

Comment:
Can the value calculated at L387 be reused? Or how about just keeping one?

Reply:
L387 doesn't record the duration by operating type (`prewrite`, `commit`, etc),
that's why we need a new metric here.

The idea of remove the origin metric crossed my mind,
but I think is not nice to simply remove it because it's depended by some panels,
We could keep both until the new panel totally replace the origin panel and the origin metric doesn't needed anymore.

Reply:
How about reusing the metrics and add extra dimensions?

Reply:
That will also dis-function the origin panels

Reply:
Why? I think prometheus is OK to be query with less dimensions.

Reply:
```
Why? I think prometheus is OK to be query with less dimensions.
```
That require we rewrite the PromQL, before that, the panel will be error.

What I mean to do is two steps (the first step may last for a while and include more than one PR, that's why I separated it into 2 steps):
1, Improve metrics, as in this PR, not touch any old metrics to keep the old panels work
2, Improve panels, in the mean time, remove the old metrics as well